corehq/apps/users/tests/test_user_data.py

```diff
@@ -129,6 +134,27 @@ def test_migrate_user_no_data(self):
         sql_data = SQLUserData.objects.get(domain=self.domain, user_id=user.user_id)
         self.assertEqual(sql_data.data, {})
 
+    def test_prime_user_data_caches(self):
+        users = [
+            self.make_commcare_user(),
+            self.make_commcare_user(),
+            self.make_commcare_user(),
+            self.make_web_user(),
+            self.make_web_user(),
```

Comment:
I'm not seeing where the Couch data is deleted on test cleanup. Looks like `delete_all_users()` is called in setup, but not during teardown.

Reply:
`make_commcare_user` and `make_web_user` call `addCleanup` on the user.